SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+This project is currently in alpha, and some features are still in development, or require further testing. However, any feedback, bug reports, or suggestions are very much welcomed.
+
+## Supported Versions
+
+The full list of Solidity language versions we support are tracked here:
```

Comment:
I think this is meant to be about slang itself, like "we will fix bugs in the versions released in the past year, but not earlier ones".

Reply:
Ah yes. Fixed.